### ISSUE TITLE
Missing required architecture armv7 in file libpv_procupine.a when archiving app for testflight (iOS)

### ISSUE THREAD

**Issue by ericn441** [Picovoice/porcupine#246]
I am running into issue when archiving the app to release for testflight via Porcupine iOS

### Expected behaviour
Archive the application and have the .ipa file generated

### Actual behaviour
I am getting an error that says i am missing instructions for armv7. When I build the app to my device. I have no issues. But once I try to archive it for testflight distribution I get the following error 
**(see attached screenshot)** 

Although removing armv7 allows me to archive the app, AppStore Connect is unable to process the build even though it can be uploaded successfully.

### Steps to reproduce the behaviour
1.) Archive the app with the armv7 architecture and see it has build errors
2.) Archive the app without armv7 and see it successfully generates an .ipa
3.) Attempt to upload the .ipa to testflight and see that it generates a processing error after the upload.

<img width="1359" alt="Screen Shot 2019-12-11 at 7 02 48 PM" src="https://user-images.githubusercontent.com/10080635/70817790-f66fa300-1d97-11ea-8bd7-876fc43e62eb.png">
<img width="1128" alt="Screen Shot 2019-12-12 at 1 02 39 PM" src="https://user-images.githubusercontent.com/10080635/70817814-fff90b00-1d97-11ea-8bd0-a72a44c7cec1.png">

**Comment by dbartle** [comment]
Porcupine does not presently have 32-bit iOS support (i.e. armv7). I suspect that you are targeting a version of iOS previous to 11 and AppStore Connect is failing because it is trying to build binaries for both 32 (iPhone 5/5C era) and 64 bit devices.

Which version of iOS are you targeting?

**Comment by ericn441** [comment]
@dbartle thanks for getting back. I set the project minimum target to be iOS 11, change the architectures to be arm64 & arm64e, it fixed the compile errors and I was able to generate and .ipa file.

Although when i upload it to TestFlight, the build appears for roughly 10 seconds then it disappears. I tried contacting Apple but they said increment the build number and try again. Which I did...And I still have the same results.

When I remove the Pico Voice library the build is able to process & be distributed. So the issue is definitely isolated to the Pico Voice library.
